umwdevlogs/F1stP0st.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Kitting is a Science
+publishDate: 2022-08-05
+excerpt: In case you were curious
+author: Ender
+draft: false
+tags: [Meaningless, Existensialism, art,]
+---
+
+# New art
+
+Check out this great fact I learned on youtube! 
+
+<img src="/images/Knitting is a science.png" width="1024" height="712" />
+
+This means something deeper I don't intend on sharing yet! but there you have it!
```

Comment:
Nitpick: same question -- should we capitalize "But"?

Reply:
your face